en/flight_modes_mc/position_slow.md

```diff
@@ -0,0 +1,69 @@
+# Position Slow Mode (Multicopter)
+
+[<img src="../../assets/site/difficulty_easy.png" title="Easy to fly" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+_Position Slow_ is a variation of the regular [Position mode](../flight_modes_mc/position.md).
+It works the exact same way but with lower maximum velocities.
+This is useful to, at the flick of a mode switch enable more precision on the stick input, ensure to not fly too quickly close to obstacles or to comply with regulations like [EASA's low-speed mode/function](https://www.easa.europa.eu/en/light/topics/flying-drones-close-people).
+
+There are three ways to provide limits for slow down.
+The vehicle can only be limited to go slower than in _Position_ mode.
+
+## Set Limits using Parameters
+
+Once the vehicle is in _Position Slow_ mode it will slow down from the usual quicker velocities of _Position_ mode down to a slower set of defaults defined by parameters.
+For example, the horizontal maximum velocity by default slows down from 10 to 3m/s and the vertical one from 3 to 1m/s.
+
+| Axis                | Position mode                                                               | Position Slow mode                   |
+| ------------------- | --------------------------------------------------------------------------- | ------------------------------------ |
+| Horizontal velocity | [MPC_VEL_MANUAL][mpc_vel_manual]                                            | [MC_SLOW_DEF_HVEL][mc_slow_def_hvel] |
+| Vertical velocity   | [MPC_Z_VEL_MAX_UP][mpc_z_vel_max_up] / [MPC_Z_VEL_MAX_DN][mpc_z_vel_max_dn] | [MC_SLOW_DEF_VVEL][mc_slow_def_vvel] |
+| Yaw rate            | [MPC_MAN_Y_MAX][mpc_man_y_max]                                              | [MC_SLOW_DEF_YAWR][mc_slow_def_yawr] |
+
+<!-- links used in table above -->
+
+[mpc_vel_manual]: ../advanced_config/parameter_reference.md#MPC_VEL_MANUAL
+[mc_slow_def_hvel]: ../advanced_config/parameter_reference.md#MC_SLOW_DEF_HVEL
+[mpc_z_vel_max_up]: ../advanced_config/parameter_reference.md#MPC_Z_VEL_MAX_UP
+[mpc_z_vel_max_dn]: ../advanced_config/parameter_reference.md#MPC_Z_VEL_MAX_DN
+[mc_slow_def_vvel]: ../advanced_config/parameter_reference.md#MC_SLOW_DEF_VVEL
+[mpc_man_y_max]: ../advanced_config/parameter_reference.md#MPC_MAN_Y_MAX
+[mc_slow_def_yawr]: ../advanced_config/parameter_reference.md#MC_SLOW_DEF_YAWR
```

Comment:
FYI only. I don't usually use this "reference link" format but will sometimes do so to make tables easier to view and edit (because when you use the link it is much smaller and so the table cells don't wrap.

Reply:
I didn't know this was possible. It's a bit like defining variables 👀

Reply:
Yes. It's useful when you've got a lot of links to the same place, because you can make them all correct in one place. But I prefer in-place links because it is easy to screw up and provide multiple definitions etc.
Worth it in tables that are so big though.